### DISCUSSION TITLE
expected 1.0.2 release timeline

### DISCUSSION THREAD

**Discussion by leonyu1010** [SixLabors/ImageSharp#1423] [Q&A] [answered]
Hi there,

just curious what is your expected 1.0.2 release timeline?

**Comment by antonfirsov** [comment, marked as answer]
The plan is to release 1.1 with a bunch of perf improvements instead of just pushing a patch release, but this will probably take us a couple of more weeks based on the open backlog points:
https://github.com/SixLabors/ImageSharp/milestone/8

Are there any post 1.0.1 fixes/improvements you are particularly interested in?

**Reply by leonyu1010** [reply]
thanks for the quick reply. 
I am using LoadPixelData to get a Image instance from byte[].  because the image is huge, I am trying the WrapMemory improvements from #1419 to avoid copying to new memory buffer.  My understanding is that  LoadPixelData copies pixel data when it creates the Image instance. 

ultimately, I need a Image instance to generate jpeg or png from my original TIFF image. 

I am still learning this cool project. maybe 1.0.1 already has a way to wrap byte[] as an Image without memory copy.

**Reply by antonfirsov** [reply]
You don't need  #1419 for that, `byte[]` is implicitly convertible to `Memory<byte>`, so you can use the existing overload [`WrapMemory<TPixel>(Configuration, Memory<TPixel>, Int32, Int32)`](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Image.html#SixLabors_ImageSharp_Image_WrapMemory__1_SixLabors_ImageSharp_Configuration_System_Memory___0__System_Int32_System_Int32_).

**Reply by antonfirsov** [reply]
Ah sorry, you still need the `Memory<byte>` -> `Memory<TPixel>` conversion. You need  #1314 which is not yet available in 1.0.1. You may find your a workaround with that PR though (`CastMemoryManager<TFrom, TTo>`).

**Reply by JimBobSquarePants** [reply]
I can do a 1.0.2 release as I fear 1.1 will take longer than we want. There's a couple of bug fixes we should ship.

**Reply by leonyu1010** [reply]
thanks for the info. 

1.0.2 would be nice, but this is not a blocker for me at the moment.  I am on 1.0.1 now without issue. thank you for maintaining this project. very cool stuff.

**Reply by JimBobSquarePants** [reply]
It's out. https://www.nuget.org/packages/SixLabors.ImageSharp/1.0.2
